Tests/Clappr_Tests/Classes/Plugins/Core/MediaControl/MediaControlTests.swift

```diff
@@ -472,19 +472,21 @@ class MediaControlTests: QuickSpec {
                         expect(MediaControlPluginMock.didCallRender).to(beTrue())
                     }
                 }
-                
-                context("when passing a plugin that is not MediaControlPlugin") {
-                    it("does not add it to the view") {
-                        let plugins = [MediaControlPluginMock(), UICorePlugin()]
+
+                context("when kMediaControlPluginsOrder is passed") {
+                    it("renders the plugins following the kMediaControlPluginsOrder order") {
+                        let pluginName = "FullscreenButton"
+                        core.options[kMediaControlPluginsOrder] = pluginName
+                        let plugins = [TimeIndicatorPluginMock(context: core), FullscreenButton(context: core)]
                         let mediaControl = MediaControl(context: core)
-                        mediaControl.mediaControlView = mediaControlViewMock
                         mediaControl.render()
 
-                        
                         mediaControl.renderPlugins(plugins)
-                        
-                        expect(mediaControlViewMock.didCallAddSubviewWithView).to(equal(plugins.first?.view))
 
+                        let bottomRightView = mediaControl.mediaControlView.bottomRight
+                        let pluginView = bottomRightView?.subviews[0]
```

Comment:
`subviews.first` instead of `subviews[0]` maybe?

Reply:
Done